cocos/3d/CCMesh.cpp

```diff
@@ -350,6 +350,7 @@ void Mesh::draw(Renderer* renderer, float globalZOrder, const Mat4& transform, u
 
 
     _meshCommand.setSkipBatching(isTransparent);
+    _meshCommand.setTransparent(isTransparent);
```

Comment:
I fixed some bugs in `Sprite3D` /  `Mesh`... perhaps @super626 needed to add that line due to my changes in `Sprite3d / Mesh`... not sure.